plugins/modules/swim_intent.py

```diff
@@ -467,11 +472,11 @@ def site_exists(self, site_name):
                 params={"name": site_name},
             )
         except Exception as e:
+            self.log("Exception occured- Site {0} doesnot exist in Cisco Catalyst Center".format(site_name), "CRITICAL")
```

Comment:
self.log("An exception occurred: Site '{0}' does not exist in the Cisco Catalyst Center".format(site_name), "CRITICAL")

Reply:
added.